src/SearchInput/SearchInput.js

```diff
@@ -257,7 +269,8 @@ SearchInput.propTypes = {
 SearchInput.defaultProps = {
     onChange: () => { },
     onEnter: () => { },
-    onSelect: () => { }
+    onSelect: () => { },
+    subStringSearch: false
```

Comment:
Boolean props default to false so we can exclude this

Reply:
other than this small change and updating with master this looks good to me! Thanks for contributing!